OpenXmlPowerTools.Tests/WmlComparerTests.cs

```diff
@@ -171,7 +171,7 @@ public void WC001_Consolidate(string testId, string originalName, string revised
                     return new WmlRevisedDocumentInfo()
                     {
                         RevisedDocument = new WmlDocument(revisedCopiedToDestDocx.FullName),
-                        Color = Color.FromName(z.Element("Color").Value),
+                        Color = ColorParser.FromName(z.Element("Color").Value),
```

Comment:
What about if the element doesn't exist? Is this verified before? Seems like some null checking is needed. It would be easiest to do `z.Element("Color")?.Value` and then do the null check in the parser.

This applies to all the similar changes below.

Reply:
@twsouthwick Do you want me to change code that was like this before ? I think it's better to keep the code external to my PR as it is and handle it in another PR, what do you think?

Reply:
Can you at least add appropriate null checking where you're touching code? I'm fine doing it in another PR, though, if you feel it will be a lot of extra noise. If you go that route, open a new issue and link it here.